libraries/resource_hab_sup.rb

```diff
@@ -49,7 +49,9 @@ def exec_start_options
           opts << "--listen-http #{new_resource.listen_http}" if new_resource.listen_http
           opts << "--override-name #{new_resource.override_name}" unless new_resource.override_name == 'default'
           opts << "--org #{new_resource.org}" unless new_resource.org == 'default'
-          opts << "--peer #{new_resource.peer}" if new_resource.peer
+          new_resource.peer.flatten.each do |peer|
```

Comment:
If `peer` is a `String`, we'll likely get a `NoMethodError` on `flatten`. We should `Array()` it. 

Quick and dirty example in `pry`:

```
[1] pry(main)> peer = "pirates"
=> "pirates"
[2] pry(main)> peer.flatten
NoMethodError: undefined method `flatten' for "pirates":String
from (pry):2:in `__pry__'
[3] pry(main)> Array(peer)
=> ["pirates"]
[4] pry(main)> peer = ["pirates", "zombies"]
=> ["pirates", "zombies"]
[5] pry(main)> peer.flatten
=> ["pirates", "zombies"]
[6] pry(main)> Array(peer)
=> ["pirates", "zombies"]
```

Reply:
So,

```ruby
Array(new_resource.peer).flatten.each do |peer|
```

We should also have a test for this that passing a string works as expected, as does passing an array. This can be done with ChefSpec.

Reply:
I actually had a PR half baked for this. the bind property already handles the problem of up-casting from a string to an array and rolling out multiple runtime flags

```
property :bind, [String, Array], coerce: proc { |b| b.is_a?(String) ? [b] : b }
...
property :bind, [String, Array], coerce: proc { |b| b.is_a?(String) ? [b] : b }
```

```
property :peer, [String, Array], coerce: proc { |b| b.is_a?(String) ? [b] : b }
...
opts.push(*new_resource.peer.map { |b| "--peer #{b}" }) if new_resource.peer
```